### PR TITLE
Feature/sploit 947 | Added Open Custom Ports plugin

### PR DESCRIPTION
**Pull request by AkhtarAmir**

### CODE REVIEW

File: plugins/aws/ec2/openCustomPorts.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            default: [80, 443]
+        }
+    },
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+        var allowed_open_ports = this.settings.open_port_allowed_list.default;
```
Comment (giorod3) [inline plugins/aws/ec2/openCustomPorts.js:26]: https://github.com/cloudsploit/scans/blob/master/plugins/aws/ec2/vpcElasticIpLimit.js

Comment (giorod3) [reply plugins/aws/ec2/openCustomPorts.js:26]: `var config = {
            vpc_elastic_ip_percentage_fail: settings.vpc_elastic_ip_percentage_fail || this.settings.vpc_elastic_ip_percentage_fail.default,
            vpc_elastic_ip_percentage_warn: settings.vpc_elastic_ip_percentage_warn || this.settings.vpc_elastic_ip_percentage_warn.default
        };`

File: plugins/aws/ec2/openCustomPorts.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+            if (!describeSecurityGroups.data.length) {
+                helpers.addResult(results, 0, 'No security groups found', region);
+                return rcb();
+            }
+
+            // Loop through each security group
+            for (var g in describeSecurityGroups.data) {
+                var group = describeSecurityGroups.data[g];
+                var resource = group.GroupId;
```
Comment (giorod3) [inline plugins/aws/ec2/openCustomPorts.js:46]: is the GroupId in arn format?

Comment (AkhtarAmir) [reply plugins/aws/ec2/openCustomPorts.js:46]: no its just an id. I'll update it to be in ARN format.

Comment (giorod3) [reply plugins/aws/ec2/openCustomPorts.js:46]: yes please, this is ready to go afterwards

**Comment by AkhtarAmir** [comment]
This branch doesn't have the latest master changes

**Pull request closed without merging**